### PR TITLE
Implemented Name on Quick Filters.

### PR DESCRIPTION
**Pull request by cbordeman**
#995

### CODE REVIEW

**Comment by rmcrackan** [comment]
I'm back in town and slowly working through my mountain of emails.

This PR looks good. I'm a little hesitant about the migration code in the QuickFilters static ctor. It's just that if anything goes wrong in a static ctor, we can get in a weird state. Historically I've handled migrations in a central place. Eg:

https://github.com/rmcrackan/Libation/blob/f7a4a95e3b2e5c0eec0b1c161da96fef567d93d6/Source/AppScaffolding/LibationScaffolding.cs#L89

If it's not too much trouble, could you do this with your migration? The next version can be 11.4.2 or 11.5 -- your choice.

RE version: a lot of users had issues last year when I updated .net. Since then, I rolled back and have been slowly adding to the code base and I *think* I fixed the issue they were having. So fairly soon, we can create an actual new release version. I see I have some other stuff from you and I'll take a look at that before creating a new release. Before releasing, I'd also like to see where that conversation goes with adding a player.

**Comment by cbordeman** [comment]
Sure, I usually use a more formal upgrade process, like you do.  I will migrate to that.

**Comment by cbordeman** [comment]
I'll use 11.5, seems like a good time.

**Comment by cbordeman** [comment]
Moved the migration.  If I understood the migration strategy a little better, I might have written the code differently, though.  It seems like if it fails, the app fails a migration method; is that what we want to happen?  Currently, if it fails, I'm creating a new, empty QuickFilters.cs...

File: Source/AppScaffolding/AppScaffolding.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net8.0</TargetFramework>
-    <Version>11.4.1.1</Version>
+    <Version>11.5</Version>
```
Comment (rmcrackan) [inline Source/AppScaffolding/AppScaffolding.csproj:5]: For stupid legacy reasons that I swear I'll fix one day (don't ask how long I've been telling myself this), please always use 4 numbers. Eg: 11.5.0.1. Don't worry if you don't get around to this; I'll fix it if you don't.

**Comment by rmcrackan** [comment]
Included in latest pre-release -- https://github.com/rmcrackan/Libation/releases/tag/v11.5.0

Let me know when you have a better idea about the player. That would be one with a big wow-factor and I'd wait for it to be finalized before making a new public release.

**Comment by cbordeman** [comment]
Occurs to me I didn't update the version # on the Winforms side.  Can you do that?

**Comment by rmcrackan** [comment]
What version on the winforms side?

**Comment by cbordeman** [comment]
Is there not a version in there somewhere?  Like in the Avalonia project file or a Properties.cs?  You're the expert.

**Comment by rmcrackan** [comment]
The only version should come from that shared spot in appscaffolding

https://github.com/rmcrackan/Libation/blob/6dcf456d06af28ba8a3bbb7874a2402832f292d5/Source/AppScaffolding/AppScaffolding.csproj#L5

**Pull request merged**